bin/sequencer_report

```diff
@@ -41,7 +40,13 @@ def first_occurence(iteratable: Iterable, predicate: Callable) -> int:
   raise ValueError()
 
 
-@dataclass(unsafe_hash=True, eq=True, order=True)
+def sanitize(string: str):
+  """Sanitize string for safe displaying"""
+  sanitized = string.replace("\n", ";")
```

Comment:
usually I'd be replacing with a "; " (space after semicolon... not sure that works here, tho)